13th-HW/Sources/Views/SettingsViewController.swift

```diff
@@ -0,0 +1,126 @@
+        tableView.delegate = self
+        return tableView
+    }()
+    
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        setupHierarcy()
+        setupLayout()
+    }
+    
+    // MARK: - Setup
+    
+    func setupView() {
+        title = "Настройки"
+        navigationController?.navigationBar.prefersLargeTitles = true
+        view.backgroundColor = .systemGray6
+    }
+    
+    func setupHierarcy() {
+        view.addSubview(tableView)
+        navigationItem.searchController = searcher
+    }
+    
+    func setupLayout() {
+        tableView.snp.makeConstraints { make in
+            make.top.right.bottom.left.equalTo(view)
+        }
+    }
+}
+
+// MARK: - DataSource extension
+
+extension SettingsViewController: UITableViewDataSource {
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        cellInstances.count
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        cellInstances[section].count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let standartCell = tableView.dequeueReusableCell(withIdentifier: StandartTableViewCell.indentifier) as? StandartTableViewCell
+        standartCell?.fillData(from: cellInstances[indexPath.section][indexPath.row])
+        standartCell?.accessoryType = .disclosureIndicator
+        
+        let userInfoCell = tableView.dequeueReusableCell(withIdentifier: UserInfoViewCell.indentifier) as? UserInfoViewCell
+        userInfoCell?.fillData(from: cellInstances[indexPath.section][indexPath.row])
+        userInfoCell?.accessoryType = .disclosureIndicator
+        
+        let withSwitchInfoCell = tableView.dequeueReusableCell(withIdentifier: WithSwitchTableViewCell.indentifier) as? WithSwitchTableViewCell
+        withSwitchInfoCell?.fillData(from: cellInstances[indexPath.section][indexPath.row])
+        withSwitchInfoCell?.selectionStyle = .none
+        withSwitchInfoCell?.accessoryType = .none
+        
+        let cellType = cellInstances[indexPath.section][indexPath.row].type
+        switch cellType {
+        case .userInfo:
+            guard let userInfoCell else { return UITableViewCell() }
+            return userInfoCell
+        case .standart:
+            guard let standartCell else { return UITableViewCell() }
+            return standartCell
+        case .withSwitch:
+            guard let withSwitchInfoCell else { return UITableViewCell() }
+            return withSwitchInfoCell
+        }
+    }
+}
+
+// MARK: - Delegate extension
+
+extension SettingsViewController: UITableViewDelegate {
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        let type = cellInstances[indexPath.section][indexPath.row].type
+        if type == .userInfo {
+            return 90
+        }
+        return 45
```

Comment:
```suggestion
        return type == .userInfo ? 90 : 45
```
а еще задавать такие жесткие значения не есть хорошо, но тут ладно